### PR TITLE
add analysis_options.yaml to dev/conductor

### PR DESCRIPTION
**Pull request by christopherfujino**
copy pasted from flutter/packages/flutter_tools/analysis_options

### CODE REVIEW

**Comment by christopherfujino** [comment]
cc @yugue

File: packages/flutter_tools/analysis_options.yaml [hunk 1]
```diff
@@ -3,6 +3,10 @@
 
 include: ../../analysis_options.yaml
 
+analyzer:
```
Comment (CaseyHillers) [inline packages/flutter_tools/analysis_options.yaml:6]: Should this be in //dev/conductor instead?

Comment (christopherfujino) [reply packages/flutter_tools/analysis_options.yaml:6]: oops, yeah, lol

Comment (christopherfujino) [reply packages/flutter_tools/analysis_options.yaml]: thanks, I was so frustrated as to why this was still failing

**Review by CaseyHillers — APPROVED** [review]

**Pull request merged**